lib/create-organization-api.ts

```diff
@@ -318,6 +329,7 @@ export default function createOrganizationApi({ http }: { http: AxiosInstance })
      * ```
      */
     getOrganizationSpaceMembership(id: string) {
+      // TODO need to move
```

Comment:
Do we need to move this method somewhere or just add it to plain client?

Reply:
It's not present yet in one of the plain endpoint objects. Let's implement on what we agreed upon and then do another round of checking whats left.

Reply:
yes, i think this can do under space memberships, getForOrganization maybe?